packages/world/src/interfaces/ISystemHook.sol

```diff
@@ -1,7 +1,13 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.0;
 
-interface ISystemHook {
+import { ERC165 } from "./ERC165.sol";
+
+// ERC-165 Interface ID (see https://eips.ethereum.org/EIPS/eip-165)
+bytes4 constant SYSTEM_HOOK_INTERFACE_ID = ISystemHook.onBeforeCallSystem.selector ^
+  ISystemHook.onAfterCallSystem.selector;
```

Comment:
I noticed `IModule` above includes `ERC165.supportsInterface.selector` but this does not. Any reason?

Reply:
good catch, should be included